### PR TITLE
feat(database): add error for unsupported backend in `IsTableNotFoundError`

### PR DESCRIPTION
**Pull request by jinghancc**
This PR modifies helper function `IsTableNotFoundError`: add error for the unsupported backend.

### CODE REVIEW

File: internal/database/dbutil/database.go [hunk 1]
```diff
@@ -15,6 +15,10 @@ import (
 	sqlite3 "modernc.org/sqlite/lib"
 )
 
+const (
+	NO_SUCH_TABLE = 1146
```
Comment (lianxmfor) [inline internal/database/dbutil/database.go:19]: maybe `MYSQL_NO_SUCH_TABLE`?

**Review by lianxmfor — APPROVED** [review]
lgtm

File: internal/database/dbutil/database.go (hunk 1 above)
Comment (jinghancc) [reply internal/database/dbutil/database.go]: fixed

**Review by YikSanChan — APPROVED** [review]
lgtm

**Pull request merged**